MPCAutofill/cardpicker/templates/cardpicker/index.html

```diff
@@ -21,27 +21,24 @@
 
     {% if mobile %}
     <div class="alert alert-primary" role="alert">
-        It seems like you're on a mobile device! Please note that MPC Autofill requires a macOS or Windows computer
+        It seems like you're on a mobile device! Please note that {{ SITE_NAME }} requires a macOS or Windows computer
         to run the executable that fills your order into MPC.
     </div>
     {% endif %}
     <div class="row">
         <div class="col-lg-6 col-md-6 col-md-12">
-            <h2>MPC Autofill <em class="orpheus" style="font-size: 1rem;">by
-                <a target="_blank" href="https://www.reddit.com/user/Chilli_Axe/">u/Chilli_Axe</a></em></h2>
+            <h2>{{ SITE_NAME }}<h2>
+            <h5>Original project by <a target="_blank" href="https://www.reddit.com/user/Chilli_Axe/">u/Chilli_Axe</a></h5>
             <p>
-                MPC Autofill automates the process of printing cards through
+                {{ SITE_NAME }} automates the process of printing playtest cards through
                 <a href="https://www.makeplayingcards.com/" target="_blank">MakePlayingCards.com</a>
-                with images hosted on public Google Drives.
+                with images hosted on community submitted Google Drives.
             </p>
             <p>
                 Type your desired card list into the text box and hit <b>Submit</b>. One card per line.
-                Click <b>Search Settings</b> to change which drives are searched, and the order in which cards are shown.
-                You can also toggle between fuzzy (forgiving) and precise (exact) search modes.
-            </p>
-            <p>
-                <b>Search for tokens</b> by prepending the query with <code>t:</code> &#8212 for example,
-                <code>t:goblin</code>.
+                Click <b>Settings</b> to change which drives are searched, and the order in which cards are shown.
+                You can also toggle between fuzzy (forgiving) and precise (exact) search mode. Search for tokens by
+                prepending the card name with <code>t:</code> &#8212 for example, <code>t:goblin</code>.

```

Comment:
let's keep the instructions for how to search for tokens on their own line here

Reply:
Done!